src/redux/web3listener.ts

```diff
@@ -22,11 +26,17 @@ const debouncedUpdateMulticallListeners = debounce(blockNumber => {
   store.dispatch(updateMulticall(blockNumber));
 }, 1000);
 
-export const web3ListenerInit = () => {
-  web3Provider.pollingInterval = 10000;
-  web3Provider.on('block', debouncedUpdateMulticallListeners);
+let typedWeb3Provider = web3Provider as JsonRpcProvider | null;
```

Comment:
ah ideally `handlers/web3` should be TS'd. I realize now that a lot of the items in `handlers` will be used even in the `redux` files so it may be beneficial to include these as part of your conversion

Reply:
I can start working on `handlers/web3` tomorrow morning!

Reply:
@jinchung `handlers/web3` ended up being relatively involved, so I opened #2511 to track that separately. Thanks!